float32[6] arm_angle     # Angle in radians for each joint
